### PR TITLE
add gitignore file

### PR DESCRIPTION
**Pull request by tchcxp**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (omgaz) [inline .gitignore]: bower_components

**Comment by camjc** [comment]
May as well add `coverage` too. and
```
# Logs
logs
*.log
npm-debug.log*
```

I think thats all we need.
(see https://www.gitignore.io/api/node for example node gitignore and desirable order.)
I didn't set this up yet because I have global gitignores set up for all this stuff.
Good idea to put it in the repo.

File: .gitignore (hunk 1 above)
Comment (camjc) [inline .gitignore]: There are none.

Comment (omgaz) [inline .gitignore]: but it has a bower.json file?

Comment (tchcxp) [inline .gitignore]: @omgaz I don't think so.

Comment (camjc) [inline .gitignore]: It definitely doesn't. No idea where you're seeing that.

**Comment by tchcxp** [comment]
@camjc @omgaz Merge please?

**Comment by camjc** [comment]
Merging! :zap:

**Pull request merged**